File: FirstBitSolutions/C Module/Assignments/Assignment 2/Ques1.c
//1. Accept two numbers from user and an operator (+,-,/,*,%) based on that perform the
//desired operations.
#include <stdio.h>
void main(){
	int a,b;
	double result;
	char operator;
	
	printf("Enter first numbers: ");
	scanf("%lf",&a);
	
	printf("Enter a operator (+,-,/,*,%): ");
	scanf(" %c",&operator);
	
	printf("Enter second numbers: ");
	scanf("%lf",&b);
	
	if(operator == '+'){
		result=a+b;
		printf("Result:%lf",result);
	}else if(operator == '-'){
		result=a-b;
		printf("Result:%lf",result);
	}else if (operator == '/'){
		if(b!=0){
			result=a/b;
			printf("Result:%lf",result);
		}else{
			printf("Division by zero is not allowed.\n");
		}
	}else if(operator == "%"){
		if(b !=0){
			result = (int)a%(int)b;
			printf("Result:%lf",result);
		}else{
			printf("Modulo  is not allowed.\n");
		}
	}else{
		printf("Invalid syntax is Entered:\n");
	}
}

File: FirstBitSolutions/C Module/CtoJava Module/Assignment 2/Ques1.java
import java.util.Scanner;
class Ques{ 
public static void main(String args[]){ 
	Scanner sc= new Scanner(System.in);
	int a,b;
	double result;
	char operator;

	System.out.print("Enter first numbers: ");
	a=sc.nextInt();

	System.out.print("Enter a operator (+,-,/,*,%): ");
	operator=sc.next().charAt(0);

	System.out.print("Enter second numbers: ");
	b=sc.nextInt();

	if(operator == '+') {
		result=a+b;
		System.out.printf("Result:%f",result);
	} else if(operator == '-') {
		result=a-b;
		System.out.printf("Result:%f",result);
	} else if (operator == '/') {
		if(b!=0) {
			result=a/b;
			System.out.printf("Result:%f",result);
		} else {
			System.out.printf("Division by zero is not allowed.\n");
		}
	} else if(operator == '%') {
		if(b !=0) {
			result = (int)a%(int)b;
			System.out.printf("Result:%f",result);
		} else {
			System.out.printf("Modulo  is not allowed.\n");
		}
	} else {
		System.out.printf("Invalid syntax is Entered:\n");
	}
	 }
 }